.buildkite/scripts/lifecycle/post_command.sh

```diff
@@ -25,8 +26,10 @@ if [[ "$IS_TEST_EXECUTION_STEP" == "true" ]]; then
   buildkite-agent artifact upload 'x-pack/test/functional/failure_debug/html/*.html'
   buildkite-agent artifact upload '.es/**/*.hprof'
 
-  echo "--- Run Failed Test Reporter"
-  node scripts/report_failed_tests --build-url="${BUILDKITE_BUILD_URL}#${BUILDKITE_JOB_ID}" 'target/junit/**/*.xml'
+  if [[ -d 'target/junit' ]]; then
```

Comment:
We generally want the failed test reporter to run, even if `target/junit` doesn't exist, so that it will catch problems that cause the junit reports to not be generated or the tests to stop running.

Maybe you could introduce an environment variable like `DISABLE_FAILED_TEST_REPORTER=true` instead?

Reply:
done 29a8bc2c4547d1e9abb29f346454ea565d0b4d4b